client/src/components/recordList.js

```diff
@@ -8,6 +8,7 @@ const Record = (props) => (
    <td>{props.record.height}</td>
    <td>{props.record.weight}</td>
    <td>{props.record.medications}</td>
+   <td>{props.record.notes}</td>
```

Comment:
This is not necessary. It should only show up on the patient's calculation page.

Reply:
Removed in most recent commit.